packages/core/admin/admin/src/components/Webhooks/EventInput/EventRow.js

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { Checkbox } from '@buffetjs/core';
+import { Checkbox, BaseCheckbox } from '@strapi/parts';
+import { upperFirst } from 'lodash';
```

Comment:
`import upperFirst from 'lodash/upperFirst'`

Reply:
Alos can you move this component inside the `pages/Webhooks/EditView` folder?

Reply:
Are you sure ? I just moved the WebhookForm inside components in order to test it